src/display/canvas.js

```diff
@@ -456,7 +448,8 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
   // Defines the number of steps before checking the execution time
   var EXECUTION_STEPS = 10;
 
-  function CanvasGraphics(canvasCtx, commonObjs, objs, imageLayer) {
+  function CanvasGraphics(canvasCtx, commonObjs, objs, imageLayer,
+                          canvasFactory) {
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99020974, please change the order of `imageLayer` and `canvasFactory` here.